warehouse/warehouse.go

```diff
@@ -770,6 +791,11 @@ func (wh *HandleT) runUploadJobAllocator() {
 			panic(err)
 		}
 
+		for _, uploadJob := range uploadJobsToProcess {
```

Comment:
Should we do this in allocation level itself? Can we not do it at handleJob level?

Reply:
@sivashanmukh  Done it here to avoid picking it up again in the allocator. Since the job might be in workerChannelMap, so its not set in progress yet